okapi-core/src/main/java/org/folio/okapi/util/DepResolution.java

```diff
@@ -784,8 +760,8 @@ static void installMaxIterations(
   /**
    * Find modules that provide a required interface.
    * @param modsAvailable all modules known
-   * @param req required interface to check.
-   * @return newest modules that meets the requirements; empty if no modules are found.
+   * @param req required interface to check
+   * @return newest modules that meets the requirements; empty if no modules are found
```

Comment:
"modules that that meet the requirements"

Reply:
Oh yes.